README.md

```diff
@@ -151,12 +180,14 @@ const App = () => (
     );
 
 export default App;
+
+//...History can now be used via other components using history.push() etc.
 ```
```

Comment:
What's the goal of this comment?

Reply:
I wanted to emphasise that by implementing redux first history the behaviour of a function such as history.push or navigate() remains the same.

Reply:
@salvoravida Is there anything else you'd like to have changed before it can be merged?